ci/travis/lib.sh

```diff
@@ -5,6 +5,38 @@ NUM_JOBS=4
 echo_red() { printf "\033[1;31m$*\033[m\n"; }
 echo_green() { printf "\033[1;32m$*\033[m\n"; }
 
+############################################################################
+# Check if the file is not present in .clangformatignore in order wheter or 
```

Comment:
wheter-> whether.
Also rephrase: "in order whether or not.." to make more sense.